pkg/reconciler/delivery/table_test.go

```diff
@@ -91,3 +132,35 @@ func Configuration(namespace, name string, co ...ConfigOption) *v1.Configuration
 	c.SetDefaults(context.Background())
 	return c
 }
+
+// this type is simply a convenient alias, see withTraffic funtion below for its purpose
+type pair struct {
```

Comment:
you can just get away with a `map[string]int64`

Reply:
Nice idea. However, I wanted to order the revisions from oldest to newest as a list, but the map would mess up that order. I think order of the traffic targets seems to matter in the Route Spec and Route Status, at least for testing purposes?